code_of_laba-5.py

```diff
@@ -0,0 +1,82 @@
+class Furniture:
+    def __init__(self, name = "", coordinates =[], size = []):
+        self.name = name
+        self.coordinates = coordinates
+        self.size = size

```

Comment:
```suggestion
class Furniture:
    def __init__(self, name="", coordinates=(), size=()):
        self.name = name
        self.coordinates = coordinates
        self.size = size
```
Because assigning a list to an argument in the __init__ method arguments list causes the argument to be mutable